pypowsybl/network.py

```diff
@@ -318,17 +323,22 @@ def write_single_line_diagram_svg(self, container_id: str, svg_file: PathOrStr)
         svg_file = _path_to_str(svg_file)
         _pp.write_single_line_diagram_svg(self._handle, container_id, svg_file)
 
-    def get_single_line_diagram(self, container_id: str) -> Svg:
+    def get_single_line_diagram(self, container_id: str, use_name: bool = False, center_name: bool = False, diagonal_label: bool = False, topological_coloring: bool = True) -> Svg:
         """
         Create a single line diagram from a voltage level or a substation.
 
         Args:
             container_id: a voltage level id or a substation id
+            use_name: (layout parameter) use names instead of ids in labels
+            center_name: (layout parameter) center labels
+            diagonal_label: (layout parameter) display diagonal labels
+            topological_coloring: (layout parameter) when False, use a nominal voltage style provider (instead of the default topological style provider)
```

Comment:
Note: we could formulate it more naturally for the python user, like "when False, coloring is based only on nominal voltage"?

Reply:
Thx for the better wording!, done.